src/requirements/graph/index.ts

```diff
@@ -0,0 +1,3 @@
+import RequirementFulfillmentGraph from './definition';
```

Comment:
what's the design decision of putting this in `definition.ts` and exporting it out of `index.ts`? Why don't you just put it in `index.ts`?

Reply:
whoops, definitely an oversight! thanks for the catch